Lib/imaplib.py

```diff
@@ -1234,13 +1234,12 @@ def _mesg(self, s, secs=None):
             sys.stderr.write('  %s.%02d %s\n' % (tm, (secs*100)%100, s))
             sys.stderr.flush()
 
-        def _dump_ur(self, dict):
-            # Dump untagged responses (in `dict').
-            l = dict.items()
-            if not l: return
-            t = '\n\t\t'
-            l = map(lambda x:'%s: "%s"' % (x[0], x[1][0] and '" "'.join(x[1]) or ''), l)
-            self._mesg('untagged responses dump:%s%s' % (t, t.join(l)))
+        def _dump_ur(self, untagged_resp_dict):
+            if not untagged_resp_dict:
+                return
+            items = ('%s: %r' % (key, value)
```

Comment:
A f-string would be welcomed here:

```suggestion
            items = (f'{key}: {value!r}'
```